donkeycar/parts/lidar.py

```diff
@@ -47,6 +80,62 @@ def shutdown(self):
         self.lidar.stop_motor()
         self.lidar.disconnect()
 
+class YDLidar(object):
+    '''
+    https://pypi.org/project/PyLidar3/
+    '''
+    def __init__(self, port='/dev/ttyUSB0'):
+        import PyLidar3
+        self.port = port
+        self.distances = [] #a list of distance measurements 
+        self.angles = [] # a list of angles corresponding to dist meas above
+        self.lidar = PyLidar3.YdLidarX4(port)
+        if(self.lidar.Connect()):
+            print(self.lidar.GetDeviceInfo())
```

Comment:
In general we started to move from `print` to `logging`. Not an issue here in particular, b/c we will have to move more `print` statements to be consistent in the future.